iroh-bytes/src/baomap.rs

```diff
@@ -179,13 +185,278 @@ pub trait Store: ReadableStore + PartialMap {
         &self,
         data: PathBuf,
         mode: ImportMode,
+        format: BlobFormat,
         progress: impl ProgressSender<Msg = ImportProgress> + IdGenerator,
-    ) -> BoxFuture<'_, io::Result<(Hash, u64)>>;
+    ) -> BoxFuture<'_, io::Result<(PinnedCid, u64)>>;
 
     /// This trait method imports data from memory.
     ///
     /// It is a special case of `import` that does not use the file system.
-    fn import_bytes(&self, bytes: Bytes) -> BoxFuture<'_, io::Result<Hash>>;
+    fn import_bytes(
+        &self,
+        bytes: Bytes,
+        format: BlobFormat,
+    ) -> BoxFuture<'_, io::Result<PinnedCid>>;
+
+    /// Set a named pin
+    fn set_tag(&self, name: Bytes, hash: Option<Cid>) -> BoxFuture<'_, io::Result<()>> {
+        let _ = name;
+        let _ = hash;
```

Comment:
is this a place holder method, or why this odd assignment?

Reply:
This is a method that does nothing. So there are two ways to make clippy happy. 1. have _name, _hash for the fn args, or do this. If you do the former then if you implement the fn it will use these names. I prefer to have the names in the signature not starting with _